src/styles/main.scss

```
@@ -1,3 +1,73 @@
+$calendar-size: 707px;
+$block-size: ($calendar-size - 7) / 7;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  margin: 10px 10px 9px;
+  width: $calendar-size;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+
+  @for $index from 1 to 31 {
+    &__month-length--#{$index} :nth-child(n + #{$index+1}) {
+      display: none;
+    }
+  }
+
+  @for $number from 1 to 32 {
+    &__day--#{$number}::before {
+      content: "#{$number}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      position: relative;
+      top: 33%;
+    }
+  }
+
+  &__day {
+    margin: 0 0 1px 1px;
```

```suggestion
    display: flex;
    gap: 1px;
```
Use **gap** instead margin